client/components/token-field/test/index.jsx

```diff
@@ -92,6 +98,14 @@ describe( 'TokenField', function() {
 	} );
 
 	beforeEach( function() {
+		var TokenFieldWrapper;
+		mockery.registerMock( 'components/tooltip', EMPTY_COMPONENT );
```

Comment:
I think we can register this on `before` instead of `beforeEach`, there's an `after` hook too where we can put the deregistration.